src/App.tsx

```diff
@@ -1,14 +1,34 @@
 /* eslint-disable max-len */
-import React from 'react';
+import React, { useContext, useEffect, useState } from 'react';
 import 'bulma/css/bulma.css';
 import '@fortawesome/fontawesome-free/css/all.css';
 
 import { TodoList } from './components/TodoList';
 import { TodoFilter } from './components/TodoFilter';
 import { TodoModal } from './components/TodoModal';
 import { Loader } from './components/Loader';
+import { Todo } from './types/Todo';
+import { getTodos, getUser } from './api';
+import { ShowModalContext } from './components/context/stateContext';
+import { User } from './types/User';
 
 export const App: React.FC = () => {
+  const [todosInitial, setTodosInitial] = useState<Todo[]>([]);
+  const [todos, setTodos] = useState<Todo[]>(todosInitial);
+  const [user, setUser] = useState<User>();
+  const [userId, setUserId] = useState(1);
```

Comment:
you don't need that here, move the Modal and try to simplify using only one state